### PR TITLE
:tada: first commit

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

File: src/components/scene/scene.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    const center = 'M250 250Z'
+
+    const getNbPath = path => path.split(/m|M/g).length - 1
+    const countMaxPath = array => array.map(getNbPath).sort((a, b) => b - a)[0]
+    const completePath = (complete, length) => (path) => {
+      const nbPath = getNbPath(path)
+      let genPath = path
+
+      for (let i = 0; i < length - nbPath; i += 1) {
+        genPath = `${genPath}${center}`
```
Comment (delphinemillet) [inline src/components/scene/scene.jsx]: use `complete` var instead of `center` const

File: src/components/scene/scene.jsx [hunk 2]
```diff
@@ -24,29 +24,29 @@ class Scene extends Component {
   attach = idx => (e) => {
     const center = 'M250 250Z'
 
-    const getNbPath = path => path.split(/m|M/g).length - 1
-    const countMaxPath = array => array.map(getNbPath).sort((a, b) => b - a)[0]
-    const completePath = (complete, length) => (path) => {
-      const nbPath = getNbPath(path)
+    const getNbShapes = path => path.split(/m|M/g).length - 1
```
Comment (fabienjuif) [inline src/components/scene/scene.jsx]: getShapesLength or getShapesCount

File: src/components/scene/scene.jsx [hunk 3]
```diff
@@ -24,29 +24,29 @@ class Scene extends Component {
...
+    const countMaxShapes = array => array.map(getNbShapes).sort((a, b) => b - a)[0]
+    const completePath = (filler, length) => (path) => {
+      const nbShapes = getNbShapes(path)
```
Comment (fabienjuif) [inline src/components/scene/scene.jsx]: shapesLength or shapesCount

File: src/components/scene/scene.jsx [hunk 4]
```diff
@@ -24,29 +24,29 @@ class Scene extends Component {
...
 
       return genPath
     }
     const normalize = draws => {
-      const maxPath = countMaxPath(flatten(draws))
-      const complete = completePath(center, maxPath)
-      const nbLayouts = draws.map(draw => draw.length).sort((a, b) => b - a)[0]
+      const maxShapes = countMaxShapes(flatten(draws))
+      const complete = completePath(center, maxShapes)
+      const nbLayers = draws.map(draw => draw.length).sort((a, b) => b - a)[0]
```
Comment (fabienjuif) [inline src/components/scene/scene.jsx]: layersCount or layersLength

File: src/components/scene/scene.jsx [hunk 5]
```diff
@@ -24,29 +24,29 @@ class Scene extends Component {
...
 
       return draws
-        // add layouts
+        // add layers
         .map(draw => {
           const newDraw = [...draw]
 
-          for (let i = 0; i < nbLayouts - draw.length; i += 1) {
+          for (let i = 0; i < nbLayers - draw.length; i += 1) {
```
Comment (fabienjuif) [inline src/components/scene/scene.jsx]: layersCount or layersLength

**Review by delphinemillet — APPROVED** [review]

**Pull request closed without merging**